src/main/java/mate/academy/util/ConnectionUtil.java

```diff
@@ -0,0 +1,29 @@
+package mate.academy.util;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+import java.util.Properties;
+
+public class ConnectionUtil {
+    public static final String DB_URL = "jdbc:mysql://localhost:3306/book_schema";
+    public static final Properties DB_PROPERTIES;
+    private static final String DRIVER = "com.mysql.cj.jdbc.Driver";
+
+    static {
+        DB_PROPERTIES = new Properties();
+        DB_PROPERTIES.put("user", "root");
+        DB_PROPERTIES.put("password", "Qwerty");
```

Comment:
let's not use literals, and make them constant

Reply:
done